docs/user_manual.md

```diff
@@ -1,70 +1,174 @@
-## Compile the library
+
+This user manual covers compiling OpenBLAS itself, linking your code to OpenBLAS,
+example code to use the C (CBLAS) and Fortran (BLAS) APIs, and some troubleshooting
+tips. Compiling OpenBLAS is optional, since you may be able to install with a
+package manager.
+
+!!! Note BLAS API reference documentation
+
+    The OpenBLAS documentation does not contain API reference documentation for
+    BLAS or LAPACK, since these are standardized APIs, the documentation for
+    which can be found in other places. If you want to understand every BLAS
+    function and definition, we recommend reading the
+    [Intel MKL reference manual](https://software.intel.com/en-us/intel-mkl/documentation)
```

Comment:
Not sure if we want MKL documentation here (which has a number of extensions that are neither in the Reference nor in OpenBLAS), AFAIK it has no advantages over the netlib one except perhaps nicer presentation ?

Reply:
sorry,had not realized that this was straight from the wiki - guess xianyi must have been in a hurry when he put together that "user manual". netlib docs usually have the added advantage of providing a nice call graph for each function